src/helpers/colors.js

```diff
@@ -0,0 +1,8 @@
+export const colors = {
+    blueFog: '#95a7b2',
+    blueFogHover: '#8d9ea8',
+    darkSky: '#424B51',
+    iceBlue: '#E9EEF4',
+    lightGrey: '#c4ced3',
+    snowWhite: '#ffffff',
```

Comment:
I'd stay with _white_ as it is pure css color. snow_ indicates it is some another shade.